Data/Sys/GameSettings/G3L.ini

```diff
@@ -1,4 +1,4 @@
-# G3LE8P - Super Monkey Ball Adventures (TM)
+# G3LE8P - Super Monkey Ball Adventure
```

Comment:
@Armada651 That's something @Linktothepast would have to look at. I don't keep track of ini stuff, so I don't know what preferences are with regards to titling.